test/activities/content/data/TestContentServiceFileEntity.js

```diff
@@ -41,7 +41,6 @@ export const contentServiceFileData = {
 		'title': 'Test Content Service File Title',
 		'url': 'https://phoenix-is-the-best.com',
 		'embedMedia': 'true',
-		'isAdvancedEditingAvailable': 'true',
 		'isAdvancedEditingEnabled': 'true',
```

Comment:
Since this field is also being removed from the siren entity, should it also be removed from this test object?